tests/transformers/test_transformers_model_export.py

```diff
@@ -2823,6 +2825,30 @@ def test_extraction_of_torch_dtype_from_pipeline(dtype):
     assert parsed == str(dtype)
 
 
+@pytest.mark.skipcacheclean
+@pytest.mark.skipif(
+    Version(transformers.__version__) < Version("4.26.1"), reason="Feature does not exist"
+)
+@flaky()
+def test_extraction_of_torch_dtype_from_underlying_model_config():
```

Comment:
Do we know of a model type that sets this parameter within the `parameters` configuration so we can test that the behavior of extraction from that means is consistent across versions of transformers (just to be on the safe side to add to this existing added test for this feature)

Reply:
Sounds great idea!